tests/python/unittest/test_numpy_op.py

```diff
@@ -2220,6 +2222,7 @@ def hybrid_forward(self, F, x):
                 assert same(ret_mx.asnumpy(), ret_np)
 
 
+@unittest.skip("Flaky test https://github.com/apache/incubator-mxnet/issues/17840")
```

Comment:
TVMOP is always turned off for GPU builds on CI now. You can remove these

Reply:
Alright, removing all the flaky test skips then